pages/linux/radeontop.md

```diff
@@ -5,16 +5,16 @@
 
 - Show the utilisation of the default AMD GPU:
 
-`sudo radeontop`
+`radeontop`
```

Comment:
I recall a thread where we discussed about removing `sudo` to a given commands.
Thoughts ?

Reply:
There have been discussions in the past about this, but per my own thoughts and [the contributing guidelines](https://github.com/tldr-pages/tldr/blob/8f9e3caf2954ba0a3da3e16047dc0eb482f0b048/CONTRIBUTING.md):

>When in doubt, keep new command-line users in mind. Err on the side of clarity rather than terseness. **For example, commands that require `sudo` should include it directly in the examples.**

Reply:
The description of this PR says:

>Remove sudo as it is not required and update color option

Per those very same guidelines, if `sudo` isn't required, it shouldn't be added.  However, if adding `sudo` makes the command run differently (better), that should be noted somewhere in the page.

Reply:
appologies, my bad.. :(

Reply:
If we're removing `sudo` from this page because it's not required but it still enhances the output, that really needs to be noted somewhre.

Reply:
Should the clause `(may require root privileges depending on your system)` be moved to the first description line so it's clearer that it applies to all of the examples?

Reply:
If it applies to all examples, then yes @aherst